src/components/sidebar-toggle-button/SidebarToggleButton.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import { injectIntl } from 'react-intl';
+
+import IconHide from '../../icons/general/IconHide';
+import IconShow from '../../icons/general/IconShow';
+import PlainButton from '../plain-button';
+import Tooltip from '../tooltip';
+
+import messages from '../../elements/common/messages';
+
+import './SidebarToggleButton.scss';
+
+type Props = {
+    isOpen: Boolean,
+    onClick?: Function,
+} & InjectIntlProvidedProps;
+
+const SidebarToggleButton = ({ intl, isOpen, onClick, ...rest }: Props) => {
+    const isCollapsed = !isOpen ? 'collapsed' : '';
+    const intlMessage = isOpen ? messages.sidebarHide : messages.sidebarShow;
+    const intlText = intl.formatMessage(intlMessage);
+    const classes = classNames({
```

Comment:
The `classNames` helper can accept a string as the first argument to avoid the `: true` in the object literal:

```
const className = classNames('bdl-SidebarToggleButton', { bdl-is-collapsed: isCollapsed });
```